src/application.js

```diff
@@ -204,6 +235,15 @@ class XtkSchemaNode {
          * @type {string}
          */
         this.nodePath = this._getNodePath(true)._path;
+        /**
+         * Element of type join have a table of join
```

Comment:
Do you mean: Element of type "link" has an array of "join" ?

Reply:
Yes change made thx